spec/suites/layer/vector/PolylineSpec.js

```diff
@@ -212,4 +212,15 @@ describe('Polyline', function () {
 			expect(polyline._latlngs).to.eql([L.latLng([1, 2])]);
 		});
 	});
+
+	describe("#style", function () {
+		it("can set weight after being added to the map", function () {
```

Comment:
It is still not completely clear to me.
1. You've stated that this PR fixes #662 (_Error when calling Path.setStyle **before** added to the map_)
2. But here you are testing it **after** adding to map.

Reply:
Hi @johnd0e 
I am not sure what you mean. I have forked leaflet to fix a bug (one line) in Path.setStyle that was introduced in version 1.5. All the test etc. are unchanged

Reply:
> All the test etc. are unchanged

I refer to 66ed2a88155699896205cb695ff508abcb8cc602 by @fodor0205